analyzer/codechecker_analyzer/analyzers/analyzer_types.py

```diff
@@ -136,6 +136,28 @@ def check_available_analyzers(analyzers, errored):
     sys.exit(1)
 
 
+def check_available_checkers(analyzer_config_map, ordered_checkers):
```

Comment:
In the test file I had a comment (https://github.com/Ericsson/codechecker/pull/3257/files#r609425261) and you wrote that it is hard to test this function because of the analyzer map. What would be if you create two functions:
```py
def get_available_checkers(analyzer_config_map: Dict[str, AnalyzerConfigHandler]) -> Set[str]:
    available_checkers = set()
    for config_handler in analyzer_config_map.values():
        available_checkers.update(config_handler.checker_groups)
    return available_checkers

def check_available_checkers(checkers: List[str], available_checkers: Set[str]):
    pass
```
This way you don't have to remove any test cases and you can test the check_available_checkers function easily.